vars.production.yml

```diff
@@ -11,8 +11,8 @@ disk_quota: 2G
 new_relic_app_name: inventory
 new_relic_monitor_mode: true
 
-routes:
-  - route: inventory.data.gov
+route_external: inventory-prod-datagov.app.cloud.gov
+route_internal: inventory-prod-datagov.apps.internal
```

Comment:
On [catalog](https://github.com/GSA/catalog.data.gov/blob/4ea89ccfa88c902fbc517d820d4bdfee295aefef/vars.production.yml#L17) it is `routes-external: catalog-prod-datagov.app.cloud.gov`, though at some later time Fuhu did add `routes-public: catalog.data.gov`. I'm not sure about the reason for that.

Reply:
That is because catalog isn't actually released to production for end users. For inventory it is, so Nick is right: the format changes, but the value of `route_external` should be `inventory.data.gov`.